packages/scoutgame-ui/src/components/common/Card/BuilderCard/BuilderCard.tsx

```diff
@@ -27,9 +39,44 @@ export function BuilderCard({
   disableProfileUrl?: boolean;
   markStarterCardPurchased?: boolean;
   type: 'default' | 'starter_pack';
+  showListButton?: boolean;
+  scoutInView?: string;
 }) {
   const isDesktop = useMdScreen();
   const isLgScreen = useLgScreen();
+  const { user } = useUser();
+  const price = builder.price;
+
+  const userListings =
```

Comment:
all this data and logic to determine a single value: lowestNonUserListing, should be done on the backend, only returning the single value to the frontend. It should just be written to the builderNft like we write currentPrice, so you don't have to calculate his on the fly all the time.

Reply:
its a bit difficult to do it from the backend as sometimes we show the scouts listing and sometimes we show your listing sometimes we show the lowest listing. So for now I'm fetching them all and selecting the one to display on the fe. That is a bit easier but we can clean it up in v2